src/components/agent-list/AgentCard/agent-card.styles.js

```diff
@@ -70,7 +71,7 @@ export const Username = styled.p`
   font-weight: bold;
   letter-spacing: 0.125rem;
   line-height: 1.375rem;
-  margin: 1.25rem 0 0.625rem 0;
+  margin: 0 0 0.625rem 0;
```

Comment:
```suggestion
  margin: 0 0 0 0;
```
and handle elsewhere?